app/channel.go

```diff
@@ -47,6 +48,63 @@ func (s *channelsWrapper) GetChannelsForTeamForUser(teamID string, userID string
 	return s.srv.getChannelsForTeamForUser(request.EmptyContext(s.srv.Log()), teamID, userID, opts)
 }
 
+func (s *channelsWrapper) GetChannelSidebarCategories(userID, teamID string) (*model.OrderedSidebarCategories, *model.AppError) {
```

Comment:
@isacikgoz do you recall why the channels APIs were accessed via the server install of app?  If we're using app then I guess the methods above should be switched as well?